group-coordinator/src/main/java/org/apache/kafka/coordinator/group/OffsetMetadataManager.java

```diff
@@ -472,7 +472,6 @@ public int deleteAllOffsets(
                 })
             );
         }
-        metrics.record(OFFSET_DELETIONS_SENSOR_NAME, records.size());
```

Comment:
do we not want to update this since we are deleting the group anyway?

Reply:
discussed with david offline; the existing behavior does not record offset deletions if it goes through deleteGroups. so keeping the existing behavior